tests/pytests/test.py

```diff
@@ -3690,7 +3690,7 @@ def test_RED_86036(env):
     for i in range(1000):
         env.cmd('hset', 'doc%d' % i, 't', 'foo')
     res = env.cmd('FT.PROFILE', 'idx', 'search', 'query', '*', 'INKEYS', '2', 'doc0', 'doc999')
-    res = res[1][4][1][7] # get the list iterator profile
+    res = res[1][1][0][9][7][0] # get the list iterator profile
```

Comment:
The `[1][1][0]` I understand, but where did the 9 and last 0 come from?

Reply:
So `[1][4]` was replaced with `[1][1][0]`, and `[1][7]` was replaced with `[9][7][0]`. I can't really tell what they were, but this is due to replacing lots of internal array pairs with a single long key-value array, to simplify the reply and align the reply structure with the `RESP3` format

Reply:
Interesting, that's quite a shift